### PR TITLE
Update: Color audit

### PR DESCRIPTION
**Pull request by ststimac**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)

Fix: #630

Went through all the CSS files and icons and updated anything that was still using the old color hex 4046dd. Also updated sections on the homepage and scanner that were still using darker blues that aligned with hex 4046dd.

### CODE REVIEW

**Comment by molant** [comment]
Looks like you deleted the `.hintrc`  file. Can you please add it back? We need it for the release process.

**Comment by ststimac** [comment]
@molant done!

**Comment by molant** [comment]
There are conflicts with `package-lock.json`. Can you please rebase with master and run `npm` again?

File: .hintrc [hunk 1]
```diff
@@ -34,4 +34,4 @@
             ]
         }
     ]
-}
+}
```
Comment (molant) [inline .hintrc]: Missing new line here. Can you please add it so there aren't any changes in this file?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@hint/configuration-web-recommended": "^6.1.0",
```
Comment (molant) [inline package.json]: Can you remove this line? Not sure why it got added

**Comment by ststimac** [comment]
@molant I think I got everything updated.

**Pull request merged**